index.html

```diff
@@ -180,7 +180,31 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
     <p class="setup-similar-label">Петр Петров</p>
   </div>
 </template>
-
+<script>
+  var fireballSize = 22;
+  var wizardSpeed = 3;
+  var wizardWidth = 70;
+
+  var getWizardHeight = function () {
+    return 1.337 * wizardWidth;
+  };
+
+  var getFireballSpeed = function (left) {
+    if (left === "true") {
+      return 5;
+    } else {
+      return 2;
+    }
+  };
+
+  var getWizardX = function (width) {
+   return width / 2 - wizardWidth / 2;
```

Comment:
```suggestion
   return (width - wizardWidth) / 2;
```
можно немного упростить